
pos embds stdev:  tensor(0.0662, grad_fn=<StdBackward0>)
Traceback (most recent call last):
  File "trials.py", line 225, in <module>
    run_trials(data_path=data_paths[i],
  File "trials.py", line 173, in run_trials
    trainer = Trainer(
  File "/chronos_data/ssmith/lib/python3.8/site-packages/transformers/trainer.py", line 602, in __init__
    self.control = self.callback_handler.on_init_end(self.args, self.state, self.control)
  File "/chronos_data/ssmith/lib/python3.8/site-packages/transformers/trainer_callback.py", line 366, in on_init_end
    return self.call_event("on_init_end", args, state, control)
  File "/chronos_data/ssmith/lib/python3.8/site-packages/transformers/trainer_callback.py", line 414, in call_event
    result = getattr(callback, event)(
AttributeError: 'WeightsAndBiasesCallback' object has no attribute 'on_init_end'
Traceback (most recent call last):
  File "trials.py", line 225, in <module>
    run_trials(data_path=data_paths[i],
  File "trials.py", line 173, in run_trials
    trainer = Trainer(
  File "/chronos_data/ssmith/lib/python3.8/site-packages/transformers/trainer.py", line 602, in __init__
    self.control = self.callback_handler.on_init_end(self.args, self.state, self.control)
  File "/chronos_data/ssmith/lib/python3.8/site-packages/transformers/trainer_callback.py", line 366, in on_init_end
    return self.call_event("on_init_end", args, state, control)
  File "/chronos_data/ssmith/lib/python3.8/site-packages/transformers/trainer_callback.py", line 414, in call_event
    result = getattr(callback, event)(
AttributeError: 'WeightsAndBiasesCallback' object has no attribute 'on_init_end'